README.md

```diff
@@ -44,6 +53,75 @@ server.on('stream', function(stream) {
 });
 ```
 
+### client
+
+```javascript
+var transport = require('spdy-transport')
+
+// NOTE: socket is some stream or net.Socket instance, may be an argument
+// of `net.createServer`'s connection handler.
+
+var client = transport.connection.create(socket, {
+  protocol: 'http2',
+  isServer: false
+})
+
+// optional for http2, however mandatory for spdy ([2, 3, 3.1])
+client.start(4)
+
+client.request({ 
+  method: 'GET',
+  host: 'localhost',
+  path: '/',
+  headers: {
+    a: 'b',
+    c: 'd'
+   }
+ }, function (err, stream) {
+  if (err) {
+    return console.log(err)
+  }
+
+  stream.on('response', function (code, headers) {
+    console.log(code, headers)
+   
+    // request body
+    stream.write(<data>)
+
+    // And other node.js Stream APIs
+    // ...
+  })
+
+  stream.on('readable', function () {
+    var chunk = stream.read()
+    if (!chunk) {
+      return
+    }
+    console.log(chunk.toString())
+  })
+})
+```
+
+### frame listener
```

Comment:
Got it. Although I believe it might be a nice reminder that a good way to check if more bytes are available is through the .on('readable') event. Have seen a couple of times folks doing something like:

```
var buf = stream.read()
if (!buf) {
  return setTimeout(tryAgainInAbit, 1e3)
}
```

What if I left only the stream.read() and a comment with a note about the .on('readable') event?